### PR TITLE
Translated code-of-conduct file

### PR DESCRIPTION
**Pull request by arshadkazmi42**
Related to #1 
-
- [x] Our Pledge
- [x] Our Standards
- [x] Our Responsibilities
- [x] Scope
- [x] Enforcement
- [x] Attribution

### CODE REVIEW

File: docs/contributing/code-of-conduct.md [hunk 1]
```diff
@@ -1,76 +1,62 @@
...
+## हमारी प्रतिज्ञा
 
-In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
-our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, gender identity and expression, level of
-experience, nationality, personal appearance, race, religion, or sexual identity
-and orientation.
+एक खुले और स्वागत वाले वातावरण को बढ़ावा देने के हित में, हम जैसे हैं
+योगदानकर्ता और अनुरक्षक हमारी परियोजना में भागीदारी करने का संकल्प लेते हैं और
```
Comment (SatyaRF) [inline docs/contributing/code-of-conduct.md]: परियोजना =>प्रोजेक्ट

File: docs/contributing/code-of-conduct.md [hunk 2]
```diff
@@ -1,76 +1,62 @@
...
-response to any instances of unacceptable behavior.
+प्रोजेक्ट मेंटेनर की ज़िम्मेदारी है की वो स्वीकार करने लायक व्यवहार का स्टैण्डर्ड स्पस्ट करें
+और उनसे किसी भी तरह के अस्वीकारिये व्यवहार के जवाब में उचित और सही कार्रवाई करने की उम्मीद है।
 
-Project maintainers have the right and responsibility to remove, edit, or reject
-comments, commits, code, wiki edits, issues, and other contributions that are
-not aligned with this code of conduct, or to ban temporarily or permanently any
-contributor for other behaviors that they deem inappropriate, threatening,
-offensive, or harmful.
+प्रोजेक्ट मेंटेनर के पास किसी भी कमेंट, कमिट, कोड, wiki बदलाव, issues, और कोई भी योगदान जो हमारी आचार संहिता की अनुसार नहीं है  उसे हटाने, बदलने या अस्वीकार करने या किसी भी योगदानकर्ता के अनुचित, धमकी, आक्रामक, या हानिकारक व्यवहार को देख कर अस्थायी या स्थायी रूप से प्रतिबंध का हक और जिम्मेदारी है।
```
Comment (SatyaRF) [inline docs/contributing/code-of-conduct.md]: Remove extra space after नहीं है

File: docs/contributing/code-of-conduct.md [hunk 3]
```diff
@@ -1,76 +1,62 @@
...
+[conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com) संपर्क करके दी जा सकती है। सभी शिकायतें ड़याँ से देखा और 
+उसकी जांच की जाएगी और उसका परिणाम आवश्यक और परिस्थितियों के लिए उपयुक्त एक प्रतिक्रिया के रूप में होगा। 
+प्रोजेक्ट टीम किसी भी घटना के रिपोर्टर के संबंध में गोपनीयता बनाए रखने के लिए बाध्य है।
+आगे प्रवर्तन नीतियों का विवरण अलग से पोस्ट किया जा सकता है।
 
-Project maintainers who do not follow or enforce the Code of Conduct in good
-faith may face temporary or permanent repercussions as determined by other
-members of the project's leadership.
+प्रोजेक्ट मेंटेनर जो नेक नीयत से आचरण संहिता का पालन या प्रवर्तन नहीं करते हैं उन पर अस्थायी या स्थायी रूप से अन्य 
+सदस्यों द्वारा निर्धारित किये गए प्रतिप्रभाव सामना करना पद सकता है।
```
Comment (SatyaRF) [inline docs/contributing/code-of-conduct.md]: पद => पड़

File: docs/contributing/code-of-conduct.md [hunk 4]
```diff
@@ -1,76 +1,62 @@
...
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at
-[conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com). All complaints will be
-reviewed and investigated and will result in a response that is deemed necessary
-and appropriate to the circumstances. The project team is obligated to maintain
-confidentiality with regard to the reporter of an incident. Further details of
-specific enforcement policies may be posted separately.
+अपमानजनक, उत्पीड़न, या अन्यथा अस्वीकार्य व्यवहार की सूचना प्रोजेक्ट टीम को यहाँ 
+[conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com) संपर्क करके दी जा सकती है। सभी शिकायतें ड़याँ से देखा और 
+उसकी जांच की जाएगी और उसका परिणाम आवश्यक और परिस्थितियों के लिए उपयुक्त एक प्रतिक्रिया के रूप में होगा।
```
Comment (SatyaRF) [inline docs/contributing/code-of-conduct.md:50]: शिकायतें ड़याँ से देखा और उसकी => शिकायतों की समीक्षा और जांच

**Review by SatyaRF — CHANGES_REQUESTED** [review]
@arshadkazmi42 I have suggested some changes. You can have a look. Thanks.

File: docs/contributing/code-of-conduct.md (hunk 4 above)
Comment (arshadkazmi42) [reply docs/contributing/code-of-conduct.md:50]: I have updated it to this 
`शिकायतें ध्यान से देखा और उसकी`

I feel this is a bit complex `शिकायतों की समीक्षा और जांच`

Any thoughts?

**Comment by arshadkazmi42** [comment]
@SatyaRF Done with fixes. Added one comment for one of the feedback

**Review by SatyaRF — APPROVED** [review]

**Comment by arshadkazmi42** [comment]
@SatyaRF Thank you for the quick review 😃

**Comment by arshadkazmi42** [comment]
@SatyaRF Can you review this PR also whenever you get time https://github.com/gatsbyjs/gatsby-hi/pull/4

Thanks

**Pull request merged**